src/index.ts

```diff
@@ -709,9 +723,56 @@ export default class IlpPluginAsymServer extends MiniAccountsPlugin {
             destination=${destination}
             error=${e && e.stack}`)
         })
+    } else if (parsedResponse.type === IlpPacket.Type.TYPE_ILP_REJECT) {
+      if (parsedResponse.data.code === 'T04') {
+        const owed = this._getAmountOwed(destination)
+        this._log.trace('sending settlement on T04 to pay owed balance.' +
+          ' destination=' + destination +
+          ' owed=' + owed.toString())
+
+        util._requestId()
+          .then((requestId: number) => {
+            const protocolData = this._sendMoneyToAccount(owed.toString(), destination)
+            this._decreaseAmountOwed(owed.toString(), destination)
```

Comment:
do we want a try/catch here that throws 'failed to create a valid claim' like the fulfill case (except without decreasing the balance)?

Reply:
There's a `.catch` after this `.then` block that will be triggered when an error occurs